Fixes custom_field issue in TalentLMS

bfreed76 opened the pull request.

markdav-is committed:
Merge branch 'develop'

bfreed76 committed:
Adds additional custom field so that updating user works in both dev and prod

markdav-is commented on TalentLMS.Api/Users.cs:
```diff
@@ -47,7 +47,8 @@ namespace Users
             string user_id,
-            string custom_field_1
+            string custom_field_1,
+            string custom_field_7
```
instead of user a record, use a dictionary, so you can swap out the name of the key.

bryceefisher committed:
Refactor user editing functionality

The method for editing a user has been refactored to accept a dictionary instead of a specific data type. Additionally, unused code related to the 'NewUser' and 'EditUser' records has been removed to clean up the file.
